_typos.toml

```diff
@@ -1,5 +1,10 @@
+[default]
+extend-ignore-re = [
+  "VALHALLA_EXTENDED_OSRM_RESPONSE.*",
```

Comment:
😂 I... did not expect this action to apply to _raw_ strings. Oh well. Sorry for any annoyance this caused!